lib/creator/generators/index.js

```diff
@@ -5,26 +5,18 @@ const Input = require('webpack-addons').Input;
 module.exports = class WebpackGenerator extends Generator {
 	constructor(args, opts) {
 		super(args, opts);
-		this.myConfig = {
-			inquirer: [],
-			config: {},
-			childDependencies: []
-		};
+		this.configuration = {};
 	}
-	Inquirer() {
-		this.myConfig.inquirer = [
-			Input('entry','What is the name of the entry point in your application?'),
-			Input('output','What is the name of the output directory in your application?')
-		];
-	}
-	Config() {
-		this.myConfig.config = {
-			entry: ' ',
-			output: ' '
-		};
+	prompting() {
+		return this.prompt([Input('entry','What is the name of the entry point in your application?'),
+			Input('output','What is the name of the output directory in your application?')]).then( (answer) => {
+				this.configuration.webpackOptions = answer;
+			});
 	}
+	config() {}
 	childDependencies() {
-		this.myConfig.childDependencies = ['webpack-addons-preact'];
+		this.configuration.childDependencies =  ['webpack-addons-preact'];
```

Comment:
Extra white space after `=`.

I'm wondering if you're skipping the pre-commit hook since it should be able to fix such issues automatically.